edb/schema/pointers.py

```diff
@@ -2343,6 +2345,13 @@ def _alter_begin(
                         context=self.source_context,
                     )
 
+                if irutils.contains_dml(self.cast_expr.ir_statement):
```

Comment:
I just realized that we can probably implement this using `ctid`. (And even without that we could do it for links, I think).
This is lower priority than fill, though, so we can push this off to 3.1 at this point, probably. File an issue to track it though.